index.html

```diff
@@ -915,37 +1027,105 @@ <h4>Text</h4>
 
         <section>
           <h4>Text Language Source</h4>
-          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating whether a <a>Text</a> object is
-            in the same language as the relevant part of the audio's
-            language (original), or if it is a representation in another language (translation):</p>
+          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating the source language of
+            a <a>Text</a> object that represents content with an inherent language.
+            If it differs from the language of the <a>Text</a> object,
+            that implies that it is a representation in another language,
+            referred to as &quot;translation&quot;;
+            conversely if it is the same that implies that the language of the <a>Text</a> is <a>Original</a>.</p>
+          <p><a>Text Language Source</a> properties SHOULD NOT be present on <a>Text</a> objects
+            that represent content without an inherent language,
+            such as untranslated descriptions of a visual scene or
+            captions representing non-dialogue sounds.</p>
           <ul>
-            <li><dfn>Original</dfn> - the <a>Text</a> is any of:
+            <li><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
               <ol>
                 <li>the same language as the dialogue that it represents in the original programme audio;</li>
-                <li>a transcription of text visible in the original programme video, in the same language as that text;</li>
-                <li>representative of non-dialogue sound and is in the <a>Primary Language</a> of the <a>DAPT Script</a>;</li>
-                <li>representative of the scene in the original programme video, and is in the <a>Primary Language</a> of the <a>DAPT Script</a>.</li>
+                <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+                <li>representative of non-dialogue sound;</li>
+                <li>a description of the scene in the programme video.</li>
               </ol>
             </li>
-            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of the Original language text in a different language.</li>
+            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of an <a>Original</a> <a>Text</a> object in a different language.</li>
           </ul>
```

Comment:
Nitpick: 
- Why is it a bulleted list? I would not use it unless we have an introductory sentence that says something like: "Based on the text language source property, text objects can be of two types:". 
- I would use the same structure as the definition of Original for Translation, that is: "Text is defined as Translation if it is a representation of an Original Text object in a different language"
- Should this (and the NOTE) rather go in the Text object definition?

Reply:
Done in [01d26df](https://github.com/w3c/dapt/pull/179/commits/01d26dfa237f62aa96d98db149b0e3fe825bd630)